src/entities/parent.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { BeforeInsert, Column, Entity, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity()
+export class Parent {
```

Comment:
Sorry for the name, it is a base entity which other entities will extend from to prevent repetition of defining shared columns amongst all entities such as "created_at", "id" , "updated_at".

This has been renamed to AbstractBaseEntity